v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -136,4 +128,81 @@ public interface KafkaReadOptions extends PipelineOptions {
   String getKafkaReadPasswordSecretId();
 
   void setKafkaReadPasswordSecretId(String value);
+
+  @TemplateParameter.GcsReadFile(
+      order = 8,
+      optional = true,
+      name = "keystoreLocation",
+      groupName = "Source",
+      parentName = "kafkaReadAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.SSL},
+      helpText =
+          "Cloud storage path for the Keystore location that contains the SSL certificate and private key.",
+      description = "Location of Keystore",
+      example = "gs://your-bucket/keystore.jks")
+  String getSourceKeystoreLocation();
+
+  void setSourceKeystoreLocation(String sourceKeystoreLocation);
+
+  @TemplateParameter.GcsReadFile(
+      order = 9,
+      optional = true,
+      name = "sourceSSL",
+      groupName = "Source",
+      parentName = "kafkaReadAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.SSL},
+      description = "Truststore File Location",
+      helpText =
+          "Location of the jks file in Cloud Storage with SSL certificate to verify identity.",
+      example = "gs://your-bucket/truststore.jks")
+  String getSourceTruststoreLocation();
```

Comment:
Should all getSrouce* params be renamed to getKafkaRead* params for consistency?